pkg/storaged/client.js

```diff
@@ -1484,4 +1494,56 @@ client.get_config = (name, def) => {
     }
 };
 
+client.in_anaconda_mode = () => !!client.anaconda;
+
+client.strip_mount_point_prefix = (dir) => {
+    const mpp = client.anaconda?.mount_point_prefix;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-084155-675f6df4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)